source/MultiGrid/MultiGrid.js

```diff
@@ -72,23 +72,28 @@ export default class MultiGrid extends Component {
     rowIndex = 0
   } = {}) {
     const { fixedColumnCount, fixedRowCount } = this.props
-
-    this._bottomLeftGrid && this._bottomLeftGrid.measureAllCells({
-      columnIndex,
-      rowIndex: rowIndex - fixedRowCount
+    const col = (columnIndex >= fixedColumnCount ? columnIndex - fixedColumnCount : columnIndex)
+    const row = (rowIndex >= fixedRowCount ? rowIndex - fixedRowCount : rowIndex)
```

Comment:
Sorry, just noticed these 2 lines. This change doesn't look right. I believe these 2 lines should be reverted to [the way it was before](https://github.com/bvaughn/react-virtualized/blob/080572f2e74af72416ba2f542ab6a6ea6aa3476a/source/MultiGrid/MultiGrid.js#L76-L91) but with your correction of my `recomputeGridSize` typo 😄